retrospective-update/cloud_logger.py

```
@@ -0,0 +1,49 @@
+            print(message_json["text"])
+            return
+
+        try:
+            response = requests.post(
+                self.webhook_post_url,
+                headers={"Content-Type": "application/json"},
+                json=message_json
+            )
+
+            if response.status_code != 200:
+                warnings.warn(f"Failed to post to webhook: {response.status_code}, {response.text}")
```

Ruff (0.12.2)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)